### PR TITLE
Add homework (07.09.23)

### PR DESCRIPTION
**Pull request by abosiy**

### CODE REVIEW

File: home_work_5/src/com/bosiy/BankAccount.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+public class BankAccount {
+
+    private String accountOwner;
+    private boolean isCredit;
+    private int accountNumber;
+    private int balance;
+
+
+    public BankAccount(String accountOwner, boolean isCredit, int accountNumber, int balance) {
+
```
Comment (scratchw0w) [inline home_work_5/src/com/bosiy/BankAccount.java]: no need for spacing

File: home_work_5/src/com/bosiy/BankAccount.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        if(!isCredit) {
+            if(balance < 0) {
```
Comment (scratchw0w) [inline home_work_5/src/com/bosiy/BankAccount.java]: move 0 to constants

File: home_work_5/src/com/bosiy/MainApplication.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.bosiy;
+
+// Task 1
+//Создать класс "Сотрудник" (Employee) с полями для имени, фамилии, возраста и зарплаты.
+// Реализовать геттеры и сеттеры для этих полей,
+// а также метод для вывода информации о сотруднике на экран.
+
+public class MainApplication {
+    public static void main(String[] args) {
+        Employee andrew = new Employee("Andrew", "Bosiy", 26, 1000);
```
Comment (scratchw0w) [inline home_work_5/src/com/bosiy/MainApplication.java:10]: add more empoloyees

**Review by scratchw0w — CHANGES_REQUESTED** [review]
Fix comments, please!

File: home_work_5/src/com/bosiy/BankAccount.java (hunk 1 above)
Comment (abosiy) [reply home_work_5/src/com/bosiy/BankAccount.java]: Fixed! :)

File: home_work_5/src/com/bosiy/BankAccount.java (hunk 2 above)
Comment (abosiy) [reply home_work_5/src/com/bosiy/BankAccount.java]: Fixed! :)

File: home_work_5/src/com/bosiy/MainApplication.java (hunk 3 above)
Comment (abosiy) [reply home_work_5/src/com/bosiy/MainApplication.java:10]: Fixed! 🥇

**Review by scratchw0w — APPROVED** [review]
Seems cool, great Job!

**Pull request merged**